templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -402,6 +406,242 @@ objects:
       ${{BROKER_AUTH}}
     caBundle: ${BROKER_CA_CERT}
 
+# CRDs for the broker.
+- apiVersion: apiextensions.k8s.io/v1beta1
+  kind: CustomResourceDefinition
+  metadata:
+    name: bundles.automationbroker.io
+  spec:
+    group: automationbroker.io
+    version: v1
```

Comment:
Is there any reason to consider these v1alpha or v1beta?

Reply:
it is not possible to have more than 1 version for an API in CRDs so it is not like we could bump up and start using v1 and still have v1beta1 around. I think because of that, just make it easy and call them v1.